compose/config/fields_schema.json

```diff
@@ -37,7 +37,15 @@
 
         "environment": {
```

Comment:
I think that would be better in a separate PR as it effects other fields. I'll look at doing that after this one has gone in.